services/middleware/README.md

```diff
@@ -0,0 +1,29 @@
+# exonum-middleware-service
+
+[![Travis Build Status](https://img.shields.io/travis/exonum/exonum/master.svg?label=Linux%20Build)](https://travis-ci.com/exonum/exonum)
+[![License: Apache-2.0](https://img.shields.io/github/license/exonum/exonum.svg)](https://github.com/exonum/exonum/blob/master/LICENSE)
+![rust 1.36.0+ required](https://img.shields.io/badge/rust-1.36.0+-blue.svg?label=Required%20Rust)
+
+`exonum-middleware-service` provides a collection of middleware
+for [Exonum blockchain framework](https://exonum.com/),
+allowing to compose Exonum transactions. For example, the service
+allows to batch transactions in order to execute the batch atomically,
+or to check the version of the service before performing a call to it.
+
+## Usage
+
+Include `exonum-middleware-service` as a dependency in your `Cargo.toml`:
+
+```toml
+[dependencies]
+exonum = "0.13"
```

Comment:
If I understand things correctly, release candidates aren't considered a full-weight releases, so the soft version requirement won't work:

```sh
cargo update
    Updating crates.io index
error: failed to select a version for the requirement `exonum = "^0.13"`
  candidate versions found which didn't match: 0.12.1, 0.12.0, 0.11.2, ...
  location searched: crates.io index
required by package `proj v0.1.0 (/Users/user/workspace/temp/proj)`
```

For a release candidate, we **must** include the full version.